### DISCUSSION TITLE
Push database changes from CLI without migration files

### DISCUSSION THREAD

**Discussion by goszczynskip** [payloadcms/payload#10105] [Q&A]
I often find myself needing to push database changes directly from the CLI without generating migration files. However, I haven't been able to find an option to do this with the current tools.

At the moment, I'm not using Drizzle migrations since I rely on Supabase migrations instead. It would be incredibly useful if there were a way to push these changes directly from the CLI.

Is this functionality currently supported, or is there a workaround for this?

**Comment by atahanyorganci** [comment]
I'm willing to implement this feature, if maintainers are interested.
